### PR TITLE
Fix the appengine/datastore sample

### PR DESCRIPTION
**Pull request by rjmooney**
This pull request fixes a few breaking issues with the sample and updates the instructions to include emulator usage.

### CODE REVIEW

**Comment by rjmooney** [comment]
Per #190, CircleCI does not test PRs, which accounts for the integration failure above.

**Comment by frankyn** [comment]
Hi @rjmooney,

Thank you for your patience for sending us a PR! I recently updated tests could you resolve conflicts and then we can move forward?

**Comment by frankyn** [comment]
friendly ping if you'd like to move forward. Closing tomorrow.

**Comment by rjmooney** [comment]
Sorry. I missed the notification. I’ll update the PR.

On Jun 19, 2018, at 5:13 PM, Frank Natividad <notifications@github.com> wrote:

friendly ping if you'd like to move forward. Closing tomorrow.

**Comment by frankyn** [comment]
Thanks!

**Comment by rjmooney** [comment]
I updated the PR but it looks like there are unrelated build issues.

File: appengine/datastore/Gemfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ source "https://rubygems.org"
 
 gem "sinatra"
 gem "google-cloud-datastore"
+gem "google-protobuf", "~>3.5.0"
```
Comment (frankyn) [inline appengine/datastore/Gemfile]: You can delete this and run `bundle update` That should resolve the issue.

**Comment by rjmooney** [comment]
Looks like there are still some issues with the CI pipeline.

**Comment by frankyn** [comment]
Apologies for the delay. @rjmooney could you merge master into your branch?

File: appengine/datastore/README.md [hunk 2]
```diff
@@ -8,12 +8,35 @@ is saved using the Dataset `save` method. Then, the ten most recent
 visits are retrieved in descending order by building a Query, and
 using the Dataset `run` method.
 
-## Run
+## Run Locally
+
+1. Make sure `gcloud` is installed.
```
Comment (frankyn) [inline appengine/datastore/README.md]: Update 1. to:

1. Install and initialize [Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts).

**Comment by tbpg** [comment]
If you merge the latest changes from master, Kokoro tests should work.

**Comment by frankyn** [comment]
Friendly bump

**Pull request closed without merging**